src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -302,14 +304,24 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
         handleHeight
         onResize={(width, height) => setImageDims({ width, height })}
       />
-      <img
-        alt="draggable background "
-        style={{
-          width: "100%",
-          ...backgroundImageStyle,
-        }}
-        src={backgroundImage}
-      />
+      {!backgroundImage ? (
+        <div className="Container__background-image-placeholder">
+          <Centered>
+            <div className="Container__background-image-placeholder_text">
+              Pick a background for your map{"\n"}(2000x1200px recommended size)
+            </div>
+          </Centered>
+        </div>
+      ) : (
+        <img
+          alt="draggable background "
+          className={classNames(
+            "Container__background-image",
+            backgroundImageClassName
```

Comment:
No need for classNames
```tsx
className={`Container__background-image ${backgroundImageClassName}`}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1995/commits/f033a214d20db26a1e8d035a115ccd345272d54f